core/src/main/java/org/elasticsearch/plugins/DiscoveryPlugin.java

```diff
@@ -52,4 +56,6 @@
     default NetworkService.CustomNameResolver getCustomNameResolver(Settings settings) {
         return null;
     }
+
+    default Set<Class<? extends ZenPing>> getZenPings(Settings settings) { return Collections.emptySet(); }
```

Comment:
If we are going to have this, then it should be documented. But I also don't see why we would have it. AFAIK, it was leftover as pluggable from when we had multicast.